bin/desi_compute_fiberflat_vs_humidity

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/env python
+
+import sys,os
+import astropy.io.fits as pyfits
+from astropy.table import Table
+import fitsio
+import numpy as np
+import glob
+from scipy.ndimage.filters import median_filter
+from scipy.signal import fftconvolve,correlate
+
+
+import matplotlib.pyplot as plt
+
+from desiutil.log import get_logger
+from desispec.io import findfile,read_fiberflat
+
+log=get_logger()
+
+year=2021
+
+wave=None
+fiberflats_in_bin=None
+bhumids_in_bin=None
+hdulist=None
+
+for spectro in range(10) :
+    cam=f"b{spectro}"
+
+    humidity_table_filename=f"humidity_table_{cam}.csv"
+    if not os.path.isfile(humidity_table_filename) :
+
+        vals=dict()
+        vals["NIGHT"]=[]
+        vals["HUMIDITY"]=[]
+
+        # loop on nights with fiberflat
+        for flat_filename in sorted(glob.glob(f"/global/cfs/cdirs/desi/spectro/redux/daily/calibnight/{year}*/fiberflatnight-{cam}-*.fits")) :
+            print(flat_filename)
+            head=fitsio.read_header(flat_filename)
+            night=head['NIGHT']
+            month=night//100
+            et_filename=f"/global/cfs/cdirs/desi/spectro/redux/daily/exposure_tables/{month}/exposure_table_{night}.csv"
+            if not os.path.isfile(et_filename) :
+                log.error(f"missing {et_filename}")
+                continue
+            et=Table.read(et_filename)
+            ii=(et['OBSTYPE']=='flat')
+            expids=list(et['EXPID'][ii])
+            #print(night,expids)
+            bhumid_vals=[]
+            for expid in expids :
+                desi_filename=findfile("raw",night=night,expid=expid)
+                #print(desi_filename)
+                spectcons=fitsio.read(desi_filename,"SPECTCONS")
+                ii=(spectcons["unit"]==spectro)
+                if np.sum(ii)==0 :
+                    log.warning(f"no spectro {spectro} in {desi_filename}")
+                    continue
+                bhumid=float(spectcons["BHUMID"][ii])
+                if bhumid<=0 :
+                    log.warning(f"unphysical humidity value = {bhumid}")
+                    continue
```

Comment:
Minor: Let's also be robust to bhumid > 100 and missing BHUMID column.

Reply:
done